src/data/meetings.json

```diff
@@ -87,5 +87,41 @@
     "remoteAudio": null,
     "localShare": null,
     "remoteShare": null
+  },
+  "remoteVideo": {
+    "ID": "remoteVideo",
+    "title": null,
+    "startTime": null,
```

Comment:
`startTime` and `endTime` are no longer part of the interface, we shouldn't add anymore

Reply:
I can remove them from all of the objects in a different PR/commit.

Reply:
sure